### PR TITLE
Show voting app between May 7 -14 (CEST)

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: src/lib/appsRegistry.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 import blockchainExplorerIcon from '../assets/icons/aepps/blockchain-explorer.svg';
 import tokenMigrationIcon from '../assets/icons/aepps/token-migration.jpg';
 
 export { default as DEFAULT_ICON } from '../assets/icons/aepps/default.svg';
 
-export const aeternityApps = [{
+const showVotingApp = inRange(
+  Date.now(),
+  (new Date('2019-05-06T22:00:00.000Z')).getTime(),
+  (new Date('2019-05-14T22:00:00.000Z')).getTime(),
```
Comment (thepiwo) [inline src/lib/appsRegistry.js]: would be okay to have it shown afterwards as well, no need to remove it I guess

Comment (thepiwo) [reply src/lib/appsRegistry.js]: Also is this timezone agnostic?

Comment (davidyuk) [reply src/lib/appsRegistry.js]: Firstly I have took ISO dates with timezones: `2019-05-07T00:00:00+0200` but appears that Date in Safari can't parse them, so I converted them to usual ISO string by `(new Date('2019-05-07T00:00:00+0200')).toISOString()` in Chrome and used them instead. ISO date strings are relative to UTC, so as I understand they are timezone agnostic.

Comment (thepiwo) [reply src/lib/appsRegistry.js]: maybe using unix timestamp would be saver and easier to work with

**Pull request merged**